android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="dev.google.cronet">
-  <!-- Flutter Boilerplate Modifications: Internet & Netwrok Permissions-->
+  <!-- Flutter Boilerplate Modifications: Internet & netwrok Permissions-->
```

Comment:
netwrok -> network (it was the spelling I meant, not the capitalization 🙈 )

Reply:
Oops :sweat: my bad.

Reply:
Update the binaries and I'll change the download url and commit both the changes togather :see_no_evil:. I less task for our CI vm.

Reply:
What do we need the jars for in the Android binaries?

Reply:
We need those jar files too :) Again, I need to refer this issue: https://github.com/unsuitable001/dart_cronet_sample/issues/3

I don't know the exact reason. :)

Reply:
I'm re-testing it on my device and I'll paste the exact error log here.

Reply:
Tried in 3 variation. Logs below.

Attempt 1: Include jar files but do not load the `.so` files from kotlin.
Output: Crashing as soon as it's getting launch. Only getting 
```Error connecting to the service protocol: failed to connect to
http://127.0.0.1:34017/YTBP04h-Lu4=/
````

Attempt 2: Remove jars but load `so` files from kotlin

```
✓ Built build/app/outputs/flutter-apk/app-debug.apk.
Installing build/app/outputs/flutter-apk/app.apk...                 8.9s
F/chromium(21501): [0804/174650.092879:FATAL:jni_android.cc(138)] Failed to find class J/N
F/chromium(21501): #00 pc 0x00000000002e09bf /data/app/dev.google.cronet_example-ZSU7uPvW02OfcK8Ob4tYpQ==/lib/arm64/libcronet.86.0.4240.198.so
F/chromium(21501): #01 pc 0x00000000002eb60f /data/app/dev.google.cronet_example-ZSU7uPvW02OfcK8Ob4tYpQ==/lib/arm64/libcronet.86.0.4240.198.so
F/chromium(21501): #02 pc 0x00000000003418af /data/app/dev.google.cronet_example-ZSU7uPvW02OfcK8Ob4tYpQ==/lib/arm64/libcronet.86.0.4240.198.so
F/chromium(21501): #03 pc 0x00000000001e1aa3 /data/app/dev.google.cronet_example-ZSU7uPvW02OfcK8Ob4tYpQ==/lib/arm64/libcronet.86.0.4240.198.so
F/chromium(21501): #04 pc 0x00000000002c1a3f /system/lib64/libart.so
F/chromium(21501): #05 pc 0x000000000000403b /system/lib64/libopenjdkjvm.so
F/chromium(21501): #06 pc 0x0000000000000bcb /system/framework/arm64/boot-core-oj.oat
F/chromium(21501): 
Error connecting to the service protocol: failed to connect to
http://127.0.0.1:37607/pQcwEtK-emc=/
```

Attempt 3: Remove jars and do not load `.so` files from kotlin

```
✓ Built build/app/outputs/flutter-apk/app-debug.apk.
Installing build/app/outputs/flutter-apk/app.apk...                 7.4s
I/OpenGLRenderer(21802): Davey! duration=906ms; Flags=1, IntendedVsync=29271908984223, Vsync=29272625650861, OldestInputEvent=9223372036854775807, NewestInputEvent=0, HandleInputStart=29272634604821, AnimationStart=29272634798155, PerformTraversalsStart=29272634807217, DrawStart=29272653090655, SyncQueued=29272657276280, SyncStart=29272657438155, IssueDrawCommandsStart=29272657732217, SwapBuffers=29272812535446, FrameCompleted=29272815417269, DequeueBufferDuration=11718000, QueueBufferDuration=765000, 
F/libc    (21802): Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0 in tid 21827 (1.ui), pid 21802 (.cronet_example)
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***         
Build fingerprint: 'asus/WW_X00TD/ASUS_X00T_2:9/PKQ1/16.2017.2009.087-20200826:user/release-keys'
Revision: '0'                                                           
ABI: 'arm64'                                                            
pid: 21802, tid: 21827, name: 1.ui  >>> dev.google.cronet_example <<<   
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0               
Cause: null pointer dereference                                         
    x0  0000000000000000  x1  00000078b2bfac78  x2  0000000000010002  x3  00000078a7281588
    x4  00000078a7281500  x5  0000007950719f88  x6  00000078a72814f0  x7  0000000000000000
    x8  0f5c9418fa297d4e  x9  0f5c9418fa297d4e  x10 0000000000000001  x11 0000000000000000
    x12 00000078a7281500  x13 00000078a72814f0  x14 0000000000004100  x15 aaaaaaaaaaaaaaab
    x16 00000078a7a6ced0  x17 000000795071b040  x18 0000000000000001  x19 00000078a7a74000
    x20 0000000000000002  x21 00000078b2bfc588  x22 00000078b1108041  x23 00000078b4755b90
    x24 00000078b1108041  x25 00000078b2b1e000  x26 00000078b2bfc588  x27 00000078a7387430
    x28 0000000000000004  x29 00000078b2bfaca0                          
    sp  00000078b2bfac50  lr  00000078a7717ca8  pc  00000078a78773a4    
backtrace:                                                              
    #00 pc 00000000003413a4  /data/app/dev.google.cronet_example-1jJaFkoD6WqfgexC4ieOzA==/lib/arm64/libcronet.86.0.4240.198.so
    #01 pc 00000000001e1ca4  /data/app/dev.google.cronet_example-1jJaFkoD6WqfgexC4ieOzA==/lib/arm64/libcronet.86.0.4240.198.so
    #02 pc 00000000002baba8  /data/app/dev.google.cronet_example-1jJaFkoD6WqfgexC4ieOzA==/lib/arm64/libcronet.86.0.4240.198.so
    #03 pc 0000000000006694  <anonymous:00000078b1080000>               
Syncing files to device ASUS X00TD...                                  ⣾

Lost connection to device.                                      
Syncing files to device ASUS X00TD...                              702ms
Oops; flutter has exited unexpectedly: "getIsolate: (112) Service has
disappeared".
⣻Oops; flutter has exited unexpectedly: "getIsolate: (112) Service has
disappeared".

```

Reply:
Ah of course, we're compiling that Kotlin helper code to a jar. Silly me. :)

Reply:
https://github.com/google/cronet.dart/releases/tag/binaries-v0.0.4

Reply:
@dcharkes Links are updated!